gradle.properties

```diff
@@ -1,5 +1,5 @@
 GROUP_ID=com.linkedin.shaky
 ARTIFACT=shaky
-VERSION_NAME=1.0.2-SNAPSHOT
```

Comment:
Shouldn't this still have SNAPSHOT for local development? Otherwise the steps in RELEASING.md don't make sense.